lib/realtime_web/controllers/tenant_controller.ex

```diff
@@ -244,6 +244,31 @@ defmodule RealtimeWeb.TenantController do
     end
   end
 
+  operation(:disconnect_clients,
+    summary: "Disconnect all clients",
+    parameters: [
+      token: [
+        in: :header,
+        name: "Authorization",
+        schema: %OpenApiSpex.Schema{type: :string},
+        required: true,
+        example:
+          "Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE2ODAxNjIxNTR9.U9orU6YYqXAtpF8uAiw6MS553tm4XxRzxOhz2IwDhpY"
+      ],
+      tenant_id: [in: :path, description: "Tenant ID", type: :string]
+    ],
+    responses: %{
+      200 => TenantHealthResponse.response(),
+      403 => EmptyResponse.response()
+    }
+  )
+
+  def disconnect_clients(%{assigns: %{tenant: tenant}} = conn, %{"tenant_id" => tenant_id}) do
+    Tenants.disconnect_clients(tenant_id)
+    PostgresCdc.stop_all(tenant, @stop_timeout)
```

Comment:
@filipecabaco this returns either `:ok` or `:error`. Shouldn't we include the correct resp code depending on the result of `PostgresCdc.stop_all/2`? That way whatever consumes the endpoint will know if something went wrong.

Reply:
Feels inconsistent when taking into account the other places this is called:

- https://github.com/supabase/realtime/blob/4770d8b6cfd1e523d3725eb6f64d0f04e71dc848/lib/realtime_web/controllers/tenant_controller.ex#L193

  - If this returns `:error` then `500` is returned.
  
- https://github.com/supabase/realtime/blob/4770d8b6cfd1e523d3725eb6f64d0f04e71dc848/lib/realtime_web/controllers/tenant_controller.ex#L242
    - Returns 204 no matter what.

I would think that for a proper disconnect upstream would want proper feedback that if failed would get `500` in response.